### PR TITLE
Defer running chart code until the dom is loaded 

### PR DESCRIPTION
**Pull request by greboid**
Doing this means chartjs can just go in the footer and be a nice single version to update

### CODE REVIEW

File: themes/default/html/analytics.html [hunk 1]
```diff
@@ -15,119 +15,122 @@
         </div>
     </div>
 
-<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.3/Chart.min.js"></script>
+<!-- Graphs -->
+<!-- Cant move this yet - it needs to be loaded before our chart script
+And i want to make that nicer before i move it to its own JS file -->
 <script>
+    document.addEventListener('DOMContentLoaded', function() { doCharts() })
```
Comment (csmith) [inline themes/default/html/analytics.html]:
```suggestion
    document.addEventListener('DOMContentLoaded', doCharts)
```

File: themes/default/html/breakdown.html [hunk 2]
```diff
@@ -31,9 +31,8 @@
 <!-- Graphs -->
 <!-- Cant move this yet - it needs to be loaded before our chart script
 And i want to make that nicer before i move it to its own JS file -->
-<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.3/Chart.min.js"></script>
 <script>
-
+    document.addEventListener('DOMContentLoaded', function() { doCharts() })
```
Comment (csmith) [inline themes/default/html/breakdown.html]:
```suggestion
    document.addEventListener('DOMContentLoaded', doCharts)
```

File: themes/default/html/item_view.html [hunk 3]
```diff
@@ -47,9 +47,8 @@ <h4><small>Time-Weighted<br>{{ mostRecent.twr_str }}%</small></h4>
 <!-- Graphs -->
 <!-- Cant move this yet - it needs to be loaded before our chart script
 And i want to make that nicer before i move it to its own JS file -->
-<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.3/Chart.min.js"></script>
 <script>
-
+    document.addEventListener('DOMContentLoaded', function() { doCharts() })
```
Comment (csmith) [inline themes/default/html/item_view.html]:
```suggestion
    document.addEventListener('DOMContentLoaded', doCharts)
```

File: themes/default/html/projections.html [hunk 4]
```diff
@@ -28,9 +28,11 @@ <h4><small>Revised Target<br>{{ revised.percent_str }}%</small></h4>
         </div>
     </div>
 
-<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.3/Chart.min.js"></script>
+<!-- Graphs -->
+<!-- Cant move this yet - it needs to be loaded before our chart script
+And i want to make that nicer before i move it to its own JS file -->
 <script>
-
+document.addEventListener('DOMContentLoaded', function() { doCharts() })
```
Comment (csmith) [inline themes/default/html/projections.html]:
```suggestion
document.addEventListener('DOMContentLoaded', doCharts)
```

File: themes/default/html/projections.html [hunk 5]
```diff
@@ -28,9 +28,11 @@ <h4><small>Revised Target<br>{{ revised.percent_str }}%</small></h4>
         </div>
     </div>
 
-<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.3/Chart.min.js"></script>
+<!-- Graphs -->
+<!-- Cant move this yet - it needs to be loaded before our chart script
```
Comment (csmith) [inline themes/default/html/projections.html:32]: Out of date now?

**Review by csmith — COMMENTED** [review]
Fly-by review :D

File: themes/default/html/projections.html (hunk 5 above)
Comment (greboid) [reply themes/default/html/projections.html:32]: No, this is (I believe) referring to moving ths <script></script> blocks to their own files and loading them from the respective theme files

**Comment by greboid** [comment]
Probably want to squash and merge this one

**Comment by greboid** [comment]
Based on a conversation on IRC.  Is it worth fixing the merge conflicts on this one or just abandoning?

**Comment by greboid** [comment]
As per conversation on IRC, you want to do this via the theme system, so closing this

**Pull request closed without merging**